c2corg_api/models/user.py

```diff
@@ -172,3 +173,24 @@ def validate_password(self, plain_password):
             'validator': colander.OneOf(default_langs)
         }
     })
+
+# Make sure that user email changes are propagated to mailing lists as well
+trigger_ddl = DDL("""
+CREATE OR REPLACE FUNCTION users.update_mailinglists_email() RETURNS TRIGGER AS
+$BODY$
+BEGIN
+  UPDATE """ + sympa_schema + """.subscriber_table
+  SET user_subscriber = NEW.email
+  WHERE user_subscriber = OLD.email;
```

Comment:
Here I would also use the user id, because this is the field used to setup the relation between table `users` and `subscriber_table`. Because the email should be unique, your solution would also work.